sway-ir/sway-ir-macros/src/lib.rs

```diff
@@ -199,20 +199,21 @@ fn pass_through_context(field_name: &Ident, attrs: &[Attribute]) -> proc_macro2:
 }
 
 fn try_parse_context_field_from_attr(attr: &Attribute) -> Option<Ident> {
-    let meta = attr.parse_meta().ok()?;
-    let Meta::List(meta_list) = meta else {
-        return None;
-    };
-    if meta_list.nested.len() != 1 {
+    let nested = attr
```

Comment:
This was a bit of a struggle but I think I got it to something reasonable. Not sure yet how to confirm that this change is safe.